cpp/src/replace/clamp.cu

```diff
@@ -251,10 +241,10 @@ struct dispatch_clamp {
   {
     CUDF_EXPECTS(lo.type() == input.type(), "mismatching types of scalar and input");
 
-    auto lo_itr         = make_pair_iterator<T>(lo);
-    auto hi_itr         = make_pair_iterator<T>(hi);
-    auto lo_replace_itr = make_pair_iterator<T>(lo_replace);
-    auto hi_replace_itr = make_pair_iterator<T>(hi_replace);
+    auto lo_itr = make_optional_iterator<T>(lo, contains_nulls::DYNAMIC{}, !lo.is_valid(stream));
+    auto hi_itr = make_optional_iterator<T>(hi, contains_nulls::DYNAMIC{}, !hi.is_valid(stream));
+    auto lo_replace_itr = make_optional_iterator<T>(lo_replace, contains_nulls::DYNAMIC{}, false);
```

Comment:
If this is always false, can't this be `contains_nulls::NO`?

Reply:
Good point, yes it can be `contains_nulls::NO`